gr-utils/python/modtool/templates/templates.py

```diff
@@ -563,26 +563,25 @@ def test_001_t(self):
   label: ...
   dtype: ...
 
-#  Make one 'inputs' list entry per input. Sub-entries of dictionary:
+#  Make one 'inputs' and one 'outputs' list entry per input/output. Keys include:
```

Comment:
I wouldn't club the input/output together in one description, because blocks can have just input or just output or none.  Perhaps this should read "Make one 'inputs' entry per input and one "outputs" entry per output"

Reply:
This is a good point. The reason for merging is mainly to prevent stating the same things twice and to reduce the "overhead". I have changed the wording as you suggest.